### DISCUSSION TITLE
How to get requestID of current Request

### DISCUSSION THREAD

**Discussion by Elitek6** [thunderclient/thunder-client-support#1297] [Q&A]
In Postman, I was able to get the ID of the current request with 

`pm.info.requestId`

Is there a way to do this in Thunder Client?

**Comment by rangav** [comment]
Hi @Elitek6 `Id` is not included in `tc` object

https://github.com/rangav/thunder-client-support/blob/master/docs/tc-types.d.ts

Can you please create a feature we will add it asap.
